### DISCUSSION TITLE
Estimated Compounded Amount incorrect spelling

### DISCUSSION THREAD

**Discussion by polkadirk** [OAK-Foundation/OAK-blockchain#496] [Automation Hub]
There's a spelling mistake on the auto-compound tab, screenshot attached below.

Additionally - this estimated compound amount is very unclear what the value is indicating to the user. I think we could consider removing it.

![image](https://github.com/OAK-Foundation/OAK-blockchain/assets/104081857/e82701aa-7417-48d7-a520-4eaaff83d3b8)

**Comment by v9n** [comment]
the text is corrected

**Comment by v9n** [comment]
this is fixed
